src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const listOfEmployees = document.querySelector('ul');
+const listOfPeople = [...document.querySelectorAll('li')];
```

Comment:
не усложняй названия переменныъ
```suggestion
const employees = document.querySelector('ul');
const people = [...document.querySelectorAll('li')];
```